build_tools/python/e2e_model_tests/run_module_utils.py

```diff
@@ -26,15 +26,26 @@ def build_run_flags_for_model(
 
 def build_run_flags_for_execution_config(
     module_execution_config: ModuleExecutionConfig,
-    gpu_id: str = "0") -> List[str]:
-  """Returns the IREE run module flags of the execution config."""
+    gpu_id: str = "0",
+    with_driver: bool = True) -> List[str]:
+  """Returns the IREE run module flags of the execution config.
+
+  Args:
+    module_execution_config: execution config.
+    gpu_id: target gpu id, if runs on GPUs.
+    without_driver: don't populate the driver flags if true. Useful for
+      generating flags for some CMake rules with a separate DRIVER arg.
+  Returns:
+    List of flags.
+  """
 
   run_flags = list(module_execution_config.extra_flags)
```

Comment:
You're doing this to copy it, right? I think `list.copy` would be clearer

```suggestion
  run_flags = module_execution_config.extra_flags.copy()
```

https://docs.python.org/3/tutorial/datastructures.html#:~:text=list%20in%20place.-,list.copy(),-Return%20a%20shallow